src/amdinfer/bindings/python/src/amdinfer/__init__.py

```diff
@@ -70,13 +70,14 @@ def stringToArray(data):
     return array.view(np.uint8)
 
 
-def ImageInferenceRequest(images, asTensor=True):
+def ImageInferenceRequest(images, asTensor=True, modelMetaData=None):
```

Comment:
In preparation for making modelMetadata required, move it as the second argument. Also double-check if this function is used anywhere already with the asTensor argument and needs to be updated

Reply:
changed and updated all other locations